### PR TITLE
Add devx-team as code owners

### PR DESCRIPTION
**Pull request by fredrik955**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,5 @@
 * @erlendvollset
 * @sanderland
+* @fredrik955
+* @polomani
+* hhenriques
```
Comment (polomani) [inline .github/CODEOWNERS]:
```suggestion
* @hhenriques
```
maybe?

**Comment by erlendvollset** [comment]
I think you can do @cognitedata/devx-team instead of listing individually

**Comment by polomani** [comment]
still don't think it's a good idea..
@cognitedata/devx-team is any of us able to actually review it properly?

**Comment by erlendvollset** [comment]
@polomani This is not a handover. I will still be involved and can answer any questions you may have. For any PR you are not confident in approving you can leave the review to me. I just think it would be good for devx to be more involved so that you can get more comfortable with the code base and doing reviews on this project.

**Comment by polomani** [comment]
@erlendvollset ok, let's start doing that

**Pull request merged**